ui/src/index.css

```diff
@@ -10,4 +10,11 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
+}
+.react-monaco-editor-container {
+  box-sizing: border-box;
+  padding: 10px 0;
+}
+.monaco .monaco-editor-background, .margin{
```

Comment:
https://stackoverflow.com/questions/44766624/custom-background-color-in-monaco-editor